src/components/screens/NewsItem.js

```diff
@@ -248,23 +86,22 @@ export const NewsItem = ({ data, navigation }) => {
           {!!logo && <Logo source={{ uri: logo }} />}
         </Wrapper>
 
-        {!!story && story}
+        {!!contentBlocks?.length &&
+          contentBlocks.map((contentBlock, index) => (
+            <StorySection
+              key={`${index}`}
```

Comment:
if its just the `index` it does not need to be stringified. we could dicuss again the topic of just having an index or more like it was before with `${index}-${contentBlock.id}`. we started to mention this in a different PR. maybe we need to do a strict research and vote for the way we want to go with from now?

Reply:
i reintroduced `contentBlock.id` as it is not harmful and in unexpected edge cases beneficial